content/TrueCommand/Administration/Systems.md

```diff
@@ -30,70 +30,77 @@ If the system has alerts or alarms, a blue circle with the number of current ale
 Each TrueNAS system listed on the **Systems** screen has an options menu that allows you to edit, delete or modify configuration settings for that system.
 Click the <i class="fa fa-ellipsis-v" aria-hidden="true" title="Options"></i> for the system to display the options menu.
 
-![Systems Options Menu](/images/TrueCommand/2.1/SystemScreenSystemActionsMenu.png "Systems Options Menu")
+![Systems Options Menu](/images/TrueCommand/2.2/SystemScreenSystemActionsMenu.png "Systems Options Menu")
 
-+ **Edit**: <i class="material-icons" aria-hidden="true" title="Configure">edit</i>
-+ **Users and Groups**: <mat-icon _ngcontent-igf-c221="" role="img" fontset="mdi" class="mat-icon notranslate mdi mdi-account-group mat-icon-no-color" aria-hidden="true" data-mat-icon-type="font" data-mat-icon-name="mdi-account-group" data-mat-icon-namespace="mdi"></mat-icon>
-+ **Update** : <i class="material-icons" aria-hidden="true" title="Update">update</i>
-+ **Launch TrueNAS Interface**: <mat-icon _ngcontent-igf-c221="" role="img" fontset="mdi" class="mat-icon notranslate mdi mdi-monitor-screenshot mat-icon-no-color" aria-hidden="true" data-mat-icon-type="font" data-mat-icon-name="mdi-monitor-screenshot" data-mat-icon-namespace="mdi"></mat-icon>
-+ **iSCSI Volumes**: <mat-icon role="img" fontset="mdi" fonticon="mdi-database" class="mat-icon mdi mdi-database mat-icon-no-color" aria-hidden="true"></mat-icon>
-+ **Services**: <mat-icon _ngcontent-igf-c221="" role="img" fontset="mdi" class="mat-icon notranslate mdi mdi-toolbox-outline mat-icon-no-color" aria-hidden="true" data-mat-icon-type="font" data-mat-icon-name="mdi-toolbox-outline" data-mat-icon-namespace="mdi"></mat-icon>
-+ **Delete**: <i class="material-icons" aria-hidden="true" title="Delete">delete</i>
-
-{{< tabs "System Control Options" >}}
-{{< tab "Edit" >}}
-Clicking the edit button <i class="material-icons" aria-hidden="true" title="Configure">edit</i> opens a panel on the right of the screen that displays the system setting fields you can edit. After making your changes, click **Save Changes** to update the system with the new values entered. 
-Click **Reset** to clear the form and reset values to the previously saved settings for the NAS.
+{{< expand "Edit" "v" >}}
+Clicking <i class="material-icons" aria-hidden="true" title="Configure">edit</i>  **Edit**  opens a panel on the right of the screen that displays the system setting fields you can edit. After making your changes, click **SAVE CHANGES** to update the system with the new values entered. 
```

Comment:
I noticed in the once tabbed setting descriptions, most all appear to start with Click rather than Clicking, so maybe this change for consistency?
```suggestion
Click <i class="material-icons" aria-hidden="true" title="Configure">edit</i>  **Edit** to open a panel on the right of the screen that displays the system setting fields you can edit. After making your changes, click **SAVE CHANGES** to update the system with the new values entered. 
```